source/task_scheduler.cpp

```diff
@@ -48,13 +48,14 @@ constexpr static int kInitErrorFailedToCreateWorkerThread = -60;
 struct ThreadStartArgs {
 	TaskScheduler *Scheduler;
 	unsigned ThreadIndex;
+	void(*ThreadStartCallback)();
 };
 
 FTL_THREAD_FUNC_RETURN_TYPE TaskScheduler::ThreadStartFunc(void *const arg) {
 	auto *const threadArgs = reinterpret_cast<ThreadStartArgs *>(arg);
 	TaskScheduler *taskScheduler = threadArgs->Scheduler;
 	unsigned const index = threadArgs->ThreadIndex;
-
```

Comment:
Super minor, but could you re-add the space between this line and the `//Clean up` line